ada/language/ast.py

```diff
@@ -7946,7 +7946,7 @@ def all_env_els_impl(seq=(Bool, True),
         doc="Returns this DefiningName's basic declaration"
     )
 
-    @langkit_property(public=True, return_type=AdaNode.entity.array,
+    @langkit_property(public=True, return_type=T.BaseId.entity.array,
```

Comment:
This should be in a separate commit

Reply:
lol sorry haha kthxbye

Reply:
Next time I will do one commit per new character